### PR TITLE
Create a fieldSet component/complement the existing one.

### PR DESCRIPTION
**Pull request by LinconOssa**
The component is created so that it can be implemented in the next activity 740 approval is expected.

### CODE REVIEW

**Review by Andresbl123 — APPROVED** [review]

File: src/components/cards/FielsetItem/FielsetItem.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { StoryFn, Meta } from "@storybook/react";
+import { Item } from ".";
+
+const story: Meta<typeof Item> = {
+  component: Item,
+  title: "components/inputs/Item",
+  decorators: [
+    (Story) => <Story />,
+    () => <Item id="123" name="Example Item" />,
```
Comment (carin91) [inline src/components/cards/FielsetItem/FielsetItem.stories.tsx:9]: This is not working properly, the id and name are inputs, defined by the controllers. Delete the decorators and create a props.ts file to include the properties and controllers for this component.

I expect that the component changes the values defined as inputs.

image

File: src/components/cards/FielsetItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+export function Item(props: ItemProps) {
+  const { id, name } = props;
+  return (
+    <StyledItem>
+      <Stack direction="column" gap="s050">
+        <Text type="label" size="medium">
+          {id}
+        </Text>
+        <Text size="medium">{name}</Text>
```
Comment (carin91) [inline src/components/cards/FielsetItem/index.tsx:17]: You must include a ternary selector to control the responsiveness of this component.
image

File: src/components/cards/FielsetItem/styles.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+import { inube } from "@inube/design-system";
+
+const StyledItem = styled.div`
+  cursor: pointer;
+  height: 104px;
+  padding: ${inube.spacing.s150};
+  background-color: ${({ theme }) =>
+    theme.color?.surface?.light?.regular || inube.color.surface.light.regular};
+  border-radius: ${inube.spacing.s100};
```
Comment (carin91) [inline src/components/cards/FielsetItem/styles.ts:10]: Refactor this. The border-radius is not defined with the spacing tokens in the other components. This defintion must be evaluated, but for now, use the standard sizes definitions (i.g: 8px).

File: src/components/cards/FielsetItem/styles.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+import { inube } from "@inube/design-system";
+
+const StyledItem = styled.div`
+  cursor: pointer;
+  height: 104px;
+  padding: ${inube.spacing.s150};
```
Comment (carin91) [inline src/components/cards/FielsetItem/styles.ts:7]: i do not understand why you are using the inube tokens for spacing but the theme/inube tokens for color.
i suggest to used the selector theme?.spacing?.s150 || inube.spacing.s150 to be consistent with the color definitions.

File: src/components/cards/FielsetItem/FielsetItem.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import { StoryFn, Meta } from "@storybook/react";
+import { Item } from ".";
+
+const story: Meta<typeof Item> = {
+  component: Item,
+  title: "components/inputs/Item",
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/components/cards/FielsetItem/FielsetItem.stories.tsx]: Resolved

File: src/components/cards/FielsetItem/FielsetItem.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { StoryFn, Meta } from "@storybook/react";
+import { Item } from ".";
+
+const story: Meta<typeof Item> = {
+  component: Item,
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/components/cards/FielsetItem/FielsetItem.stories.tsx]: Resolved

File: src/components/cards/FielsetItem/FielsetItem.stories.tsx (hunk 1 above)
Comment (LinconOssa) [reply src/components/cards/FielsetItem/FielsetItem.stories.tsx]: Resolved

File: src/components/cards/FielsetItem/index.tsx (hunk 2 above)
Comment (LinconOssa) [reply src/components/cards/FielsetItem/index.tsx]: Resolved

File: src/components/cards/FielsetItem/styles.ts (hunk 3 above)
Comment (LinconOssa) [reply src/components/cards/FielsetItem/styles.ts]: Resolved

File: src/components/cards/FielsetItem/styles.ts (hunk 4 above)
Comment (LinconOssa) [reply src/components/cards/FielsetItem/styles.ts]: Resolved

**Pull request merged**